kotlin-runtime/ftl-runtime/src/test/kotlin/xyz/block/ftl/schemaextractor/ExtractSchemaRuleTest.kt

```diff
@@ -16,12 +18,20 @@ import kotlin.test.AfterTest
 
 @KotlinCoreEnvironmentTest
 internal class ExtractSchemaRuleTest(private val env: KotlinCoreEnvironment) {
+  @BeforeEach
```

Comment:
here's where we add those `testdata/dependencies` files to the classpath

Reply:
Nice!